### DISCUSSION TITLE
How to use Two-way Control

### DISCUSSION THREAD

**Discussion by rbrose** [nolimits4web/swiper#4466] [Q&A] [answered]
- Swiper Version: ^6.5.7
- Platform/Target and Browser Versions: Windows, ANDROID, CHROME, ETC.

## What You Did

I want controller 2 swiper together. If i swipe top swiper, then the secound swiper should be swipe, too.
Here an example how it works with ngx-swiper-wrapper:

[stackblitz ngx-swiper-wrapper](https://stackblitz.com/edit/ngx-swiper-2-controller?file=projects/app/src/app/app.component.html)

And here the Angular 11 with Swiper Version 6.5.7:

[stackblitz Swiper Angular 11](https://stackblitz.com/edit/swiper-angular-2-controller?file=src/app/app.component.html)

## Expected Behavior

Want controller both swiper together.

## Actual Behavior

Can't access controller.control

**Comment by rbrose** [comment, marked as answer]
Ok it works now :-)

https://stackblitz.com/edit/swiper-angular-2-controller?file=src/app/app.component.html

**Reply by ghost** [reply]
A little late but I couldn't get my two swipers to control each other. Your example on stackblitz helped a lot so thank you for sharing it! ;)
